libraries/AP_Spektrum_Telem/AP_Spektrum_Telem.cpp

```diff
@@ -0,0 +1,784 @@
+/*
+
+   Inspired by work done here
```

Comment:
this is actually copied from AP_Frsky_Telem, the inspired by here makes no sense

Reply:
Ok

Reply:
Fixed